bot/resources/tags/dict-get.md

```diff
@@ -0,0 +1,15 @@
+Often while using dictionaries in Python, you may run into `KeyErrors`. This error is raised when you try to access a key that isn't present in your dictionary.\
+While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.\
```

Comment:
This shouldn't contain a backslash since we want to have a line break
```suggestion
While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.
```